ci/pipelines/perf-tests.yml

```diff
@@ -145,13 +145,13 @@ definitions:
         params:
           command: scale
           app_name: pay-stubs
-          instances: 1
+          instances: ((.:service-volumes.scale_down_to.stubs))
```

Comment:
Where is this `service-volumes` variable being loaded from?
Also pretty neat that we can define variables in yml files and load them like this.

Reply:
Yeah, it's really cool :)

I do wish they had a set-var task as well as load, sadly not, but this is pretty neat still!